reports/helmcharts/helm-chart-versions.sh

```diff
@@ -130,21 +129,47 @@ for chart in $(echo "$charts" | jq -c '.[]'); do
   fi
 
   # Enhance document with additional information
+  helm_chart_name=$(echo "$chart" | jq -r '.name')
   uuid=$(uuidgen)
   created_on=$(date '+%Y-%m-%d %H:%M:%S')
+  id="${cluster_name}-${helm_chart_name}"
+
 
   document=$(echo "$chart" | jq --arg cluster_name "$cluster_name" \
                                 --arg verdict $verdict \
-                                --arg id "$uuid" \
+                                --arg id "$id" \
                                 --arg environment "$environment" \
                                 --arg created_on "$created_on" \
                                 --arg report_type "table" \
                                 --arg display_name "HELM Repositories" \
                                 --arg color_code $color_code '. + {id: $id, environment: $environment, createdOn: $created_on, lastUpdated: $created_on, displayName: $display_name, cluster: $cluster_name, verdict: $verdict, colorCode: $color_code, reportType: $report_type}')
 
   documents+=("$document")
+  
 done
 
+cosmosdb_account_name="sds-platform-version-reporter"
+cosmosdb_database_name="reports"
+cosmosdb_container_name="helmcharts"
```

Comment:
These are also hardcoded in the save-to-cosmos.py script but it has the option for overrides using environment variables:
```python
database = os.environ.get("COSMOS_DB_NAME", "reports")
container_name = os.environ.get("COSMOS_DB_CONTAINER", "helmcharts")
```

Its probably better that you set this as variables in your script and have the values come from those environment variables. You would need to add them to the Dockerfile as well.